repository/src/main/java/org/pentaho/platform/repository2/unified/webservices/IUnifiedRepositoryWebService.java

```diff
@@ -67,6 +67,10 @@ RepositoryFileDto createFolderWithAcl( final String parentFolderId, final Reposi
   @Deprecated
   List<RepositoryFileDto> getChildren( final String folderId );
 
+  default void doesFileExist( String path ) throws Exception {
```

Comment:
why are you making this interface function `default` when no other function is?

i understand the technical purpose of '`default`` keyword